src/field_theory/minpoly/gcd_monoid.lean

```diff
@@ -54,6 +54,48 @@ begin
   { exact (monic hs).map _ }
 end
 
+/- TODO: see if the `no_zero_smul_divisors S L` assumption can be lifted -/
+theorem is_integrally_closed.map_minpoly_eq_minpoly_fraction_ring [no_zero_smul_divisors S L] :
+  minpoly K (algebra_map S L s) = (map (algebra_map R K) (minpoly R s)) :=
+begin
+  --a few "trivial" preliminary results to set up the proof
```

Comment:
Can some of these results be extracted as separate lemmas?

Reply:
I've extracted one as a separate lemma, and I'm not sure the rest are worth extracting from the proof